### PR TITLE
Handle an uri as key of a grantee

### PR DESCRIPTION
**Pull request by kirikiriyamama**
A grantee `LogDelivery` has only `uri`, so I'd like to handle an `uri` as key of a grantee.

### CODE REVIEW

**Comment by kirikiriyamama** [comment]
I don't understand how to update `doc/_resource_types/s3.md`. Can you tell me? :bow:

**Comment by k1LoW** [comment]
rename s3.md to s3_bucket.md (I forgot) and write some.
and `bundle exec ./lib/awspec/bin/toolbox docgen > doc/resource_type.md`

**Comment by kirikiriyamama** [comment]
I got it wrong :bow: I've just updated doc.

File: lib/awspec/generator/spec/s3_bucket.rb [hunk 1]
```diff
@@ -22,9 +22,10 @@ def generate_grant_specs(acl)
       end
 
       def grant_linetemplate
-        template = <<-'EOF'
-it { should have_acl_grant(grantee: '<%= grant.grantee.display_name %>', permission: '<%= grant.permission %>') }
-EOF
+        grantee = 'grant.grantee.display_name || grant.grantee.uri || grant.grantee.id'
```
Comment (k1LoW) [inline lib/awspec/generator/spec/s3_bucket.rb]: :+1:

**Comment by k1LoW** [comment]
LGRM!

**Comment by k1LoW** [comment]
Released as v0.24.2.

**Comment by kirikiriyamama** [comment]
:pray:

**Pull request merged**